cli/commands/terraform/hook.go

```diff
@@ -169,3 +171,12 @@ func executeTFLint(ctx context.Context, terragruntOptions *options.TerragruntOpt
 	}
 	return nil
 }
+
+func cloneTerragruntOptionsForHook(opts *options.TerragruntOptions, hookName string) *options.TerragruntOptions {
+	newOpts := *opts
+	newOpts.Env = util.CloneStringMap(opts.Env)
+	newOpts.Env["TERRAFORM_COMMAND"] = opts.TerraformCommand
```

Comment:
Let's make these environment variables more generic.

e.g.
`TG_CTX_COMMAND`
`TG_CTX_HOOK_NAME`

The motivation behind requesting that they're named like this is that we want them to definitely avoid collisions with user defined environment variables (users should know that they should never set `TG_CTX_` prefixed variables if they don't want the values clobbered by Terragrunt), and it avoids using terminology specific to Terraform, when that might be confusing, given that the user might be wrapping OpenTofu instead.

Reply:
Agree, renamed. Thanks for the suggestion!